src/Libraries/DynamoUnits/Symbol.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace DynamoUnits
+{
+    /// <summary>
+    /// A text symbol used to associate a value with a Unit system.
+    /// </summary>
+    public class Symbol
```

Comment:
I think that is an oversight.  I will fix it.  The original reason was because of the implementation of the deserialization logic and binding for the UI nodes for SelectedItems.  The out of the box dropdown did not have the same issue as the other converter or input node but I think the logic should be the same.